### ISSUE TITLE
Avoid self-referential descriptions 

### ISSUE THREAD

**Issue by IngridT1** [redhat-documentation/supplementary-style-guide#42]
I think that we need to add this admonition to the Supplementary Style Guide: Avoid self-referential descriptions. We have them all over our guides, and they are content focused, not customer focused. Anytime you write something like this, you're taking the customer out of their mindset and forcing them to think like us. 

DITA Best Practices (https://www.amazon.com/DITA-Best-Practices-Roadmap-Architecting/dp/0132480522) has great guidance on this:

> Avoid Writing Short Descriptions That Are Self-Referential
> No one likes a narcissist, so avoid short descriptions that start with language that makes the topic the focus of the short description. For example, avoid the following phrasing:
> 
> • “This topic describes....”
> 
> • “This concept covers....”
> 
> • “This information is about....”
> 
> • “The following chapter explains....”
> 
> These constructions take up valuable space without adding any useful information.

> **Incorrect**
> Mountain bike wheel comparison
> This topic compares 26-inch mountain bike wheels
> to 29-inch wheels.
> 
> **Correct**
> Mountain bike wheel comparison
> Deciding which wheel size to use for your mountain
> bike depends on what environments you ride in. The 26-inch
> wheel and the 29-inch wheel both have advantages and disad-
> vantages depending on where and how you ride.
>

**Comment by bergerhoffer** [comment]
+1 for adding guidance to avoid this. Not that we probably need a comprehensive list, but I see "This section..." a lot. But same deal.

I quick glance through the IBM style guide and I couldn't find any guidance on this either.

The only thing that I'd like further clarification on is introductory sentences for reference modules. I feel like there are going to be lots of examples like "The following table describes..." and would like to know if that is also to be avoided, or if it's okay.

**Comment by inoxx03** [comment]
@IngridT1 +1 to this suggestion. thank you for raising it. I think we could  definitely use less of _This guide explains how you can do X_ in our product documentation. If we don't make it a rule, we could at least make it  as a recommended practice to avoid  using such statements in our product docs:)
@bergerhoffer To your point: I keep seeing self-referential opening paragraphs being used throughout several sets of product docs for Middleware. Most commonly I spot these in module introductions. Can't say, I approve of this particular stylistic quirk:)

I've done a cursory search of the IBM style guide and a search of other online resources and language blogs to see if there are any formal rules or recurring (sensible) arguments being made against using self-referential opening sentences and I have found none (yet).

However, I concur that we should probably avoid using self-referential statements in module introductions, chiefly because they add to the volume of text without providing commensurate information value.

**Comment by IngridT1** [comment]
@inoxx03 The DITA Best Practices guide is the resource that you've been looking for! Just like the IBM Style Guide, it is published by IBM, so it has authority. DITA is just the IBM term for the kind of modular documentation that we are currently doing. 

When the DITA guide says "topic," you can substitute our term "module." When DITA says "short description," think "introduction to a module." 

I can name two good reasons to avoid self-referential language:

- Anytime you start using phrases like "the module," "the chapter," "the topic," etc., you're forcing users to think about the book, the chapter, or the topic instead of about solving their problems or learning how to do something. In short, self-referential descriptions ruin the user experience of our documentation.

- Phrases like "the section" or "the guide" take up valuable space and add nothing to the conversation.

**Comment by IngridT1** [comment]
@bergerhoffer I confess that I am not sure about phrases like "The following table describes . . ." "This diagram shows . . .". It could be that we should just have a number and caption for every table and that would take care of some of the issues (for example, "Table 1. Parameters to use in a configuration file" or something like that). Then, you can use the table number to refer to the table.

**Comment by bergerhoffer** [comment]
@IngridT1 So I assume that we'd still want an introductory sentence/paragraph to reference modules. My question was, if it's a reference module containing a table, is a intro sentence saying "This table describes ...", or should we avoid that? And if so, then what would be some good example intros for reference modules.

I can't imagine that we'd only want to have a table title/caption with no intro sentence. Also, I don't necessarily think that it would be a good idea to use the table number as a reference, since those are auto generated in AsciIDoc and could change if another table is added.

**Comment by oraNod** [comment]
+1 for avoiding self-referential text in abstracts. My 2c is that this PR shouldn't be about eliminating phrases but developing better writing habits and approaching content from the user's perspective. You should always focus on user goals at higher level content. 

That said I think it's perfectly valid to introduce pieces or groups of reference content with phrases like "This/the following table/diagram describes ..." These phrases aid navigation for scanning, especially for users with assistive technology such as screen readers.

I think it's important to mix it up and avoid excessive "stand alone" introductory sentences. By stand alone, I mean the only purpose of the sentence in the module is to introduce reference content. I often like to lead in with conceptual explanation then go for a phrase like "..., as in the following diagram:"

At times I reference the procedure directly, especially if it's something that demonstrates a principle of application behaviour for learning but not something you would do in a production environment. "This procedure shows you how to ..."

**Comment by stoobie** [comment]
This all sounds good in theory. There are problematic exceptions that arise, or if they are not exceptions, they are more complicated than most scenarios. Of course I can't think of any off the top of my head to present as an example, but I know that just last week I ran into such a situation. I can try to locate the example -- maybe I just needed a suggestion on how to deal with it.

In the case of tables or figures, I think that a description can be very helpful, together with a caption: "Figure A describes blah blah", the same way that a seemingly obvious screenshot requires explanation. If presenting a figure or table out of the blue requires the user to put 1+1 together to figure out why that figure or table is there, than an explanation should be added. If it's blatantly obvious, than perhaps not, but again, "blatantly obvious" is in the eyes of the beholder.

**Comment by IngridT1** [comment]
@bergerhoffer @stoobie When I looked at the DITA Best Practices guidelines again, they seem to be focused on short descriptions--the introductory info at the start of a module--I think that some people are calling this an "abstract". With that context, you should be able to say, "This table . . ." or "Figure 1.1 . . .". In fact, the DITA Best Practices book does use that language more than once.

As far as reference modules go, I would not begin the reference module with "This table of commands shows . . ." (or similar), but would start with a sentence that describes what the commands do and then refer to the table. 

@oraNod Sure, we should all be better writers and more customer-focused. But some of how we learn to do that is by having specific guidelines that we are trying to follow. A guideline is easier to follow that a blanket statement, "Be customer focused."

**Comment by stoobie** [comment]
@IngridT1 You know, I was thinking that you could make your minimalism classes a lot more minimal if you would just tell people "Write gooder!"

**Comment by oraNod** [comment]
@IngridT1 Not just about being customer focused but avoiding lazy writing. "This topic is about" is a shortcut and approaches the content from the writer's perspective, not the audience perspective. I feel like that should be part of the guideline description or something so it doesn't make it sound like the guideline is about banning a particular construction.

**Comment by leswilliams44** [comment]
Hi, @IngridT1 and @bergerhoffer:

Is this issue ready to be voted on during our next meeting?

**Comment by IngridT1** [comment]
Yes, @leswilliams44 I think so.

**Comment by bergerhoffer** [comment]
@IngridT1 would you be up for drafting a PR with the recommendation? I don't think that there have been any objections to adding some guidance on this.

**Comment by IngridT1** [comment]
Sure, @bergerhoffer, I can do that. Do we want to talk about it in our meeting or just here?

**Comment by bergerhoffer** [comment]
@IngridT1 I don't think it's necessary to discuss at the meeting since it isn't a controversial topic. Once the PR is open, then folks have a chance to weigh in if anything needs to be adjusted. Thanks!

**Comment by IngridT1** [comment]
@bergerhoffer @cbudz What part of the guide does this belong in? General Formatting? Or should we make a new section that this can be part of?

**Comment by IngridT1** [comment]
I've written the blurb, but I'm not sure where to insert it.

**Comment by bergerhoffer** [comment]
@IngridT1 Yeah I think that under "General formatting" should be good enough for now. If we have more items to add in the future that look like we could create a new section appropriate for this too, we can always move it then.

**Comment by bergerhoffer** [comment]
@IngridT1 you mentioned that you had a blurb written for this, can you submit it as a PR?
